Source/EasyNetQ/SubscriptionResult.cs

```diff
@@ -5,11 +5,11 @@ namespace EasyNetQ
 {
     public sealed class SubscriptionResult : ISubscriptionResult
```

Comment:
Maybe it is worth doing `SubscriptionResult` as `readonly struct` too and delete `ISubscriptionResult` interface? `SubscriptionResult` is the only implementation.

Reply:
Done